sending-jwt-upstream/README.md

```diff
@@ -0,0 +1,138 @@
+# Sending JWT Claims Upstream
+
+Extending the example [JWT Access Control](/jwt-access-control/README.md) we now want to send certain JWT claims to the backend protected by Couper's access control.
+
+In the following example we use tokens and keys from https://jwt.io/. 
+
+First, we take the configuration file from the "JWT Access Control" example:
+
+```hcl
+server "secured-api" {
+  access_control = ["JWTToken"]
+  api {
+    endpoint "/private/**" {
+      path = "/**"
+      backend {
+        origin = "https://httpbin.org/"
+      }
+    }
+  }
+
+}
+
+definitions {
+  jwt "JWTToken" {
+    header = "Authorization"
+    signature_algorithm = "RS256"
+    key = <<EOF
+-----BEGIN PUBLIC KEY-----
+MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDGSd+sSTss2uOuVJKpumpFAaml
+t1CWLMTAZNAabF71Ur0P6u833RhAIjXDSA/QeVitzvqvCZpNtbOJVegaREqLMJqv
+FOUkFdLNRP3f9XjYFFvubo09tcjX6oGEREKDqLG2MfZ2Z8LVzuJc6SwZMgVFk/63
+rdAOci3W9u3zOSGj4QIDAQAB
+-----END PUBLIC KEY-----
+        EOF
+  }
+}
+```
+
+Now we go to https://jwt.io/, switch the Algorithm to "RS256" and copy the content of the first textarea in the "VERIFY SIGNATURE" box in the right ("Decoded") column.
```

Comment:
i didn't get it right on first read.  i wanted to copy the key from couper conf to jwt.io…

maybe we should start the initial example without a key:

```
 // copy from jwt.io
 key = "…"
```

Reply:
The initial example is from the jwt-access-control example and already has a key.